src/components/Scaffolding/Scaffolding.jsx

```diff
@@ -22,30 +22,33 @@ import {
     return input.replace(/[^a-z0-9\s]/gi, '').replace(/(gt)/gi, '')
   }
   const Hit = ({ hit }) => {
-  
     return (
       <article className="hit-card">
-        <a target="_blank" rel="noopener noreferrer"
-          href={hit.repository && hit.repository.url ? hit.repository.url: '/'}>
-          <div className="product-wrapper">
-              <div className="product-summary">
-                <div className="product-title">
-                <h3>{hit.name}</h3> 
-                </div>
-                <div className="product-avatar">
-                  <img src={hit.owner.avatar} height="40" width="40"/>
-                </div>
-              </div>
-              <div className="product-name">
-              <h3>{hit.owner.name}</h3>
-              </div>
-              
-              <div className="product-desc">
-                <p>{htmlDecode(hit.description)}</p>
-              </div>
+       {
+          hit && (
```

Comment:
Hit already has the metadata you need, no need for hard coded value

Reply:
It doesn't have the package link, so I'm using the name from the metadata to get to the package.